### PR TITLE
Updated the learn page design

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
This PR updates the learn page design and fixes a few small issues.

**NOTE**
The links for cards and buttons are missing.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
There is a build error in the `layer5-ng` branch. Still trying to figure out what is causing this :grimacing:

**Comment by Nikhil-Ladha** [comment]
I found the issue, let's see if that works.

**Review by Tanuj22 — COMMENTED** [review]
- At around 1173px and 1000px the meshery logo is covering the text a little. Can be fixed by decreasing the z-index ig. 
-  At around 1031px the workshop cards are covering the text a little. 
- From 799px the text should be be center and the button at the bottom? 
Screenshot from 2021-01-16 19-00-49
- Can you set a max-width for the sections inside `Container` so that it doesn't spread out too much.
Screenshot from 2021-01-16 19-03-04

File: src/sections/Learn/learnpage.style.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import Istio_Book from "../../assets/images/learn/istio-book-cover-frame.png";
+
+export const LearnPageWrapper = styled.div`
+    .page-header-section {
```
[the remark this answers is not in the document]
Comment (Nikhil-Ladha) [reply src/sections/Learn/learnpage.style.js:5]: Indeed. I will make that in a seperate PR. If that's, fine.

**Comment by Nikhil-Ladha** [comment]
I can update that but the resolutions at which these issues are occurring are not generic one's. Like, that's happening for some random resolution and for a small range. Don't you think that won't bother?

**Comment by Nikhil-Ladha** [comment]
For the workshops, section I didn't `center` aligned the text purposefully. Trying something different 😅. Does it look fine or should it be `center` aligned only?

**Comment by Nikhil-Ladha** [comment]
> * Can you set a max-width for the sections not inside `Container` so that it doesn't spread out too much.
>   ![Screenshot from 2021-01-16 19-03-04](https://user-images.githubusercontent.com/42486853/104813197-a6320280-582d-11eb-8b67-1cb3ecb79a43.png

Hmmm...this is awkward..... @leecalcote does it look so weird in your 4k monitor as well 😆....or is it just due to changing resolution on dev tools?

**Comment by Tanuj22** [comment]
> I can update that but the resolutions at which these issues are occurring are not generic one's. Like, that's happening for some random resolution and for a small range. Don't you think that won't bother?

Yeah makes sense

**Comment by Tanuj22** [comment]
> For the workshops, section I didn't `center` aligned the text purposefully. Trying something different 😅. Does it look fine or should it be `center` aligned only?

Wasn't really sure myself 😅 . @leecalcote @Jashpatel1 thoughts?

**Comment by leecalcote** [comment]
> Hmmm...this is awkward..... @leecalcote does it look so weird in your 4k monitor as well 😆....or is it just due to changing resolution on dev tools?

_ahem_ a 5k display 😁. 


Here is an issue with text overlapping (this was an issue before this PR). // @kishore007k 

Screen Shot 2021-01-17 at 7 45 18 PM

A couple items to attend to (these might have been called out previously, I'm not sure):

Screen Shot 2021-01-17 at 7 49 19 PM

**Review by Tanuj22 — APPROVED** [review]

**Pull request merged**